src/collective/solr/profiles/default/registry.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<registry>
+  <records interface="collective.solr.interfaces.ISolrSchema"
+           prefix="collective.solr">
+    <value key="active">True</value>
```

Comment:
@tomgross Solr needs to be disabled by default. Otherwise installing c.solr on Plone site creation won't work.